### DISCUSSION TITLE
Assertion Error

### DISCUSSION THREAD

**Discussion by Hemiptelea** [coqui-ai/TTS#2124] [General Q&A] [answered]
assert (x > 1).sum() + (x < -1).sum() == 0

in the vits.py, the error AssertionError becomes.
what should I do?

And the code above seems to be the code for audio normalization. 
@erogol

**Comment by Hemiptelea** [comment, marked as answer]
by normalize my dataset, it worked

**Reply by ABSOLUTE-ZER0** [reply]
@queechy, can you elaborate what did you normalize exactly? Im a bit new to all this, so any help would be appreciated. I'm getting the following assert error while trying to train ljspeech vit.

Should I normalize audio files? If so how did you achieve that? Im using the Mozilla Common Voice database

**Reply by Hemiptelea** [reply]
i used normalize() in pydub

In detail, load audio file to AudioSegment(in pydub) and normalize and export

I think the below link will help you

https://stackoverflow.com/questions/42492246/how-to-normalize-the-volume-of-an-audio-file-in-python

**Reply by ABSOLUTE-ZER0** [reply]
Thanks a ton, worked like a charm!!
